src/ctelnet.cpp

```diff
@@ -1342,6 +1343,27 @@ void cTelnet::postMessage( QString msg )
                 if( body.size() )
                     mpHost->mpConsole->print( body.join('\n').append('\n'), 150, 0, 0, 0, 0, 0 );  // Red on black
             }
+            else if( prefix.contains("LUA") )
+            {
+                mpHost->mpConsole->print( prefix, 210, 90, 60, 0, 0, 0 ); // Orangeish on black
+                mpHost->mpConsole->print( firstLineTail.append('\n'), 80, 160, 255, 0, 0, 0 );  // Blue on black
+                for( quint8 _i = 0; _i < body.size(); _i++ )
+                {
+                    QString temp = body.at(_i);
+                    temp.replace('\t', "        ");
+                    body[_i] = temp.rightJustified( temp.length() + prefixLength );
+                }
+                body.replaceInStrings("Lua error:", "");
+                if( body.size() > 1 )
+                {
+                    mpHost->mpConsole->print( body.takeFirst().append('\n'), 50, 200, 50, 0, 0, 0 );  // Green on black
+                    mpHost->mpConsole->print( body.join('\n').append('\n'), 200, 50, 50, 0, 0, 0 );  // Red on black
+                }
+                else if( body.size() )
+                {
+                    mpHost->mpConsole->print( body.join('\n').append('\n'), 50, 200, 50, 0, 0, 0 );  // Green on black
+                }
+            }
```

Comment:
I was initially going to climb all over an addition to the `postMessage()` code but you do seem to have done a good job of plumbing it in and handling the more than two line case {body.size() > 1 at the point where you test it} and then the more than two lines case {body.size() != 0}. :+1: